### ISSUE TITLE
Script

### ISSUE THREAD

**Issue by Stargazer284** [Stargazer284/Bot-cani-Terrier#2]
import discord
import numpy as np
from PIL import Image
from tensorflow.keras.models import load_model
from io import BytesIO

TOKEN = "TOKEN"

model = load_model("keras_model.h5", compile=False)

with open("labels.txt", "r", encoding="utf-8") as f:
    class_names = f.readlines()

intents = discord.Intents.default()
intents.message_content = True

client = discord.Client(intents=intents)

@client.event
async def on_ready():
    print(f"Bot online come {client.user}")

@client.event
async def on_message(message):
    if message.author == client.user:
        return

    if not message.attachments:
        return

    attachment = message.attachments[0]

    if not attachment.filename.lower().endswith((".png", ".jpg", ".jpeg", ".webp")):
        return

    image_bytes = await attachment.read()
    image = Image.open(BytesIO(image_bytes)).convert("RGB")
    image = image.resize((224, 224))

    image_array = np.asarray(image).astype(np.float32)
    normalized_image_array = (image_array / 127.5) - 1

    data = np.ndarray(shape=(1, 224, 224, 3), dtype=np.float32)
    data[0] = normalized_image_array

    prediction = model.predict(data)

    index = np.argmax(prediction)
    class_name = class_names[index].strip()
    confidence = prediction[0][index]

    await message.channel.send(
        f"Riconosco: **{class_name}**\n"
        f"Confidenza: **{confidence * 100:.1f}%**"
    )

client.run(TOKEN)
